src/models/UserSettings.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Class to store all user settings.
+ */
+class UserSettings {
+  /**
+   * Default constructor.
+   * @param {object} [props] - Object that contains all properties to set.
+   * @param {string} [props.displayType] - Display type for diagrams, only 'grid' or 'table' are
+   * accepted.
```

Comment:
Start a new sentence from "only" and break line before.

Reply:
nope

Reply:
ugly